src/widgets/TreeDataCard/ui/components/MenuRow.tsx

```
@@ -72,33 +54,39 @@ export const MenuRow: FC<MenuRowProps> = ({ node }) => {
-            )}
-            {node.onDelete && (
-              <>
-                <Menu.Separator />
-                <Menu.Item value="delete" onClick={node.onDelete}>
-                  <PiTrash />
-                  <Box flex={1}>Delete</Box>
-                </Menu.Item>
-              </>
+            {node.menu.map((item) =>
+              item.children ? (
+                <Menu.Root positioning={{ placement: 'right-start', gutter: 2 }}>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)